src/features/onboarding/register/StepPayment.tsx

```diff
@@ -31,7 +31,7 @@ const StepPaymentMethod = (props: TStepPaymentMethodProps): JSX.Element => {
       method: PaymentMethods.PslAddress,
     },
     {
-      name: 'Airdrop “Promo Code”',
+      name: 'Pastel Promocode',
       method: PaymentMethods.AirdropPromoCode,
```

Comment:
name is Pastel Promocode, but method is still AirdropPromoCode - seems to be a mistake.
@ngvtuan is working on Pastel promo code in parallel pull request, so I suggest to just undo this change